installation/system-requirements.md

```diff
@@ -1,42 +1,46 @@
 slug: system-requirements
 tags: .net maui, ui for .net maui, telerik .net maui, macOS, windows, system requirements, visual studio, visual studio code
 position: 1
 ---
 
 # System Requirements for Telerik UI for .NET MAUI
 
->note Telerik .NET MAUI 3.2.0 supports .NET 7.0. The Telerik .NET MAUI 7.1.0 version supports .NET 9.0 Preview 6. For creating a project targeting a .NET 9 Preview, you need to use a Visual Studio 2022 Preview. 
+You can develop applications with Telerik UI for .NET MAUI on both Windows and macOS. Depending on your OS and development preferences, you can use either Visual Studio 2022 17.8 (or later), or the latest Visual Studio Code with the .NET MAUI extension.
 
-> Microsoft discontinued .NET 7 support in May 2024. As of 2024 Q4 (November), Telerik .NET MAUI ends the support for .NET 7. Upgrade your apps to .NET 8.
-> By upgrading to the most recent version of .NET, you will leverage the latest features, performance enhancements, and security improvements offered by both Microsoft and Telerik.
-> For more information about the supported .NET versions in .NET MAUI, please review the <a href="https://dotnet.microsoft.com/en-us/platform/support/policy/maui" target="_blank">.NET MAUI Support Policy.</a>
+## Supported Target Platforms
 
-To develop applications with Telerik UI for .NET MAUI, you need to install Visual Studio 2022 17.8 or greater, or the latest Visual Studio Code with the .NET MAUI extension.
+You can develop applications with Telerik UI for .NET MAUI for the following platforms:
 
-### Visual Studio
+| Target Platform | Supported Version |
+| ------------- | --------------- |
+| Android | 5.0 (API 21) or higher |
+| iOS | 11 or higher |
+| macOS | 11 or higher, using Mac Catalyst |
+| Windows* | 11 and Windows 10, version 1809 or higher, using Windows UI Library (WinUI) 3 |
 
-Install Visual Studio 2022 17.8 to develop applications with Telerik UI for .NET MAUI on Windows by following the [installation steps on MS Learn](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=vswin#installation-1).
+*The macOS operating system does not support the development of WinUI apps for the Windows platform.
 
-### Visual Studio Code
+## Supported .NET Versions  
 
-Install the latest Visual Studio Code to develop applications with Telerik UI for .NET MAUI on Windows or macOS by following the [installation steps on MS Learn](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code#installation-2).
+* Telerik UI for .NET MAUI 3.2.0 supports .NET 7.0. 
 
-## Supported Platforms
+    > Microsoft discontinued .NET 7 support in May 2024. Telerik UI for .NET MAUI ends the support for .NET 7 in November 2024. Upgrade your apps to .NET 8 to leverage the latest features, security, and performance enhancements.
```

Comment:
we have to remove this, if the revamp sections will be live for the release, 
if we release the updates earlier, then we ca keep this information.